pandas/core/dtypes/common.py

```diff
@@ -1386,7 +1386,7 @@ def is_bool_dtype(arr_or_dtype) -> bool:
         # we don't have a boolean Index class
         # so its object, we need to infer to
         # guess this
-        return arr_or_dtype.is_object and arr_or_dtype.inferred_type == "boolean"
+        return arr_or_dtype.is_object() and arr_or_dtype.inferred_type == "boolean"
```

Comment:
wow! did this change anything? (w/o the rest of the patch)

Reply:
Afaik, no other tests were failing after I changed this. I am running all tests on master with only this change just to make sure.

Reply:
Perhaps I should do a coverage check to see which tests covers this line.

Reply:
Tests were green.